Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
>i'm saying shouldn't delete it if it exists

expected:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1426/files#diff-57e53faa7c8496e4e9ff143e74804f78R54

input:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1426/files#diff-57e53faa7c8496e4e9ff143e74804f78R90

this one is ready to go IMO